seatunnel-engine/seatunnel-engine-client/src/main/java/org/apache/seatunnel/engine/client/ConnectorInstanceLoader.java

```diff
@@ -31,16 +31,24 @@
 
 import org.apache.seatunnel.shade.com.typesafe.config.Config;
 
+import com.google.common.collect.Lists;
+import org.apache.commons.lang3.tuple.ImmutablePair;
+
+import java.net.URL;
+import java.util.List;
+
 import scala.Serializable;
 
```

Comment:
Utility class should not be instantiated, it's better to add
```
private ConnectorInstanceLoader() {
    throw new IllegalStateException("Utility class");
  }
```

Reply:
done